internal/resume/templates/latex.go

```diff
@@ -25,6 +26,12 @@ func latexEscape(s string) string {
 	return s
 }
 
+func latexCensor(s string) string {
+  re := regexp.MustCompile(`\|\|(.*)\|\|`)
```

Comment:
I just ran some tests by adding multiline strings to my resume YAML, but the rendered output seems to ignore the newlines.
Otherwise there weren't any issues I found with the censoring, it acted as if there were no newlines basically.